syndicate/connection/api_gateway_connection.py

```diff
@@ -209,8 +214,49 @@ def create_method(self, api_id, resource_id, method,
             params['requestParameters'] = request_parameters
         if request_models:
             params['requestModels'] = request_models
+        if request_validator:
+            params['requestValidatorId'] = self.create_request_validator(
+                api_id, request_validator)
         self.client.put_method(**params)
 
+    def create_request_validator(self, api_id, request_validator):
+        """
+        Helper function to create a request validator. Returns its id.
+
+        :type api_id: str
+        :param api_id: Identifier of the associated RestApi.
+        :type request_validator: dict
+        :param request_validator: Dictionary with values to create request
+        validator. It could contains name of validator and validate parameters
+        (validate_request_body, validate_request_parameters or both)
+        :return: str, identifier of created RequestValidator.
+        """
+        try:
+            validator_name = request_validator.pop('name')
+        except KeyError:  # not critical error
+            validator_name = None
+
+        request_validator_params = {}
+
+        if ('validate_request_body', True) in request_validator.items():
+            request_validator_params.update({'validateRequestBody': True,
+                                             'name': 'Validate body'})
+        if ('validate_request_parameters', True) in request_validator.items():
+            request_validator_params.update({'validateRequestParameters': True,
+                                             'name': 'Validate query string '
+                                                     'parameters and headers'})
+
+        if not validator_name and len(request_validator_params) == 3:
```

Comment:
Please do not use "magic numbers". Init some constant var with a human-readable description instead of simple number 3.